src/SelectableBox/index.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Form from '../Form';
+import SelectableBoxSet from './SelectableBoxSet';
+import { useCheckboxSetContext } from '../Form/FormCheckboxSetContext';
+import { useRadioSetContext } from '../Form/FormRadioSetContext';
+
+const INPUT_TYPES = [
+  'radio',
+  'checkbox',
+];
+
+const SelectableBox = ({
```

Comment:
Should this be wrapped with `React.forwardRef` so consumers can pass a ref if needed?

Reply:
Right, component is wrapped